src/components/checkbox/index.tsx

```diff
@@ -197,13 +209,21 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
     }
   };
 
+  getErrorColor = () => !this.state.isValid && Colors.$textDanger;
```

Comment:
There are two colors for this state: `$outlineDanger` and `$textDangerLight`.

Reply:
fixed